src/test/java/org/springframework/samples/petclinic/owner/OwnerTest.java

```diff
@@ -63,4 +64,36 @@ public void setPetsInternalgetPetsInternalTest() {
 		assertEquals(pet, onlyPet);
 	}
 
+	@Test
+	public void getPetTest() {	
+		Pet pet = new Pet();
+		pet.setName("Pochi");
+		instance.addPet(pet);
+		
+		assertEquals(pet, instance.getPet("Pochi"));	
+		assertEquals(pet, instance.getPet("Pochi", false)); //tests pet object exists
+		assertEquals(null, instance.getPet("Pochi", true));	//tests pet object doesn't exist
```

Comment:
I think these should all be separate test cases, rather than having them all together. In other words, maybe have a "testThatPetExists" and "testThatPestDoesntExist" test case or something along those lines.

Reply:
Ok fixed.